src/PrefsManager.cpp

```diff
@@ -237,6 +237,7 @@ PrefsManager::PrefsManager() :
 	m_ShowDancingCharacters		( "ShowDancingCharacters",		SDC_Random ),
 	m_bUseUnlockSystem		( "UseUnlockSystem",			false ),
 	m_fGlobalOffsetSeconds		( "GlobalOffsetSeconds",		-0.008f ),
+	m_fDefaultGroupOffsetSeconds	( "DefaultGroupOffsetSeconds",		0.0f ),
```

Comment:
Kinda think we should discuss this a bit.  I feel like we should keep it null and change the default to itg later? Just cause this would make people's stuff out of sync out the box if it defaults to itg at first lol.  It'd be double itg syncs for most content itg players are playing rn.  Keeping it null means nothing changes by default on upgrading and folks have time to make the change? That's just my thought tho I'm not super pressed about it but can def see ppl being bothered that they **have** to re-download or resync their content now as opposed to everything past, future,  and present continuing to work as they were. Like anyone who individually syncs songs (i.e. dimo iirc) would need to resync those songs or re-download them all. Also on top of that,  no packs in the past had group.ini's so now if it defaults itg rn, any pack that wasn't null offset would be offsync by itg standards. (A pack that had the 9ms applied to every song individually would have it applied twice now cause since there is no group.ini it defaults to getting an additional 9ms).


I think we should default to itg once we have nine or null stuff in.

Reply:
I think my interpretation was that defaulting to ITG implies "packs on this machine already have the implicit 9ms bias applied, so we can automatically remove the bias to move them to null internally".

I think you interpreted defaulting to ITG as "apply the 9ms offset to every unmarked song on the machine" which is not the same.

My interpretation meant that everything is always handled as null internally and just determines whether or not we need to internally adjust the offset or not.

Reply:
Definition matters. Currently we can interpret this value ambiguously

Reply:
Right that's why I think just writing the definition in line above the preference will help. Defaulting to ITG with my interpretation already pushes setups towards null. The other interpretations pushes non-marked charts towards ITG using the ITG bias which is not what we want to be doing.